python/py-fuzzer/fuzz.py

```diff
@@ -152,13 +152,16 @@ def print_description(self, index: int, num_seeds: int) -> None:
 
 def fuzz_code(seed: Seed, args: ResolvedCliArgs) -> FuzzResult:
     """Return a `FuzzResult` instance describing the fuzzing result from this seed."""
+    # TODO(carljm) remove once we debug the `ty` hang on this seed
+    skip_check = seed == 120
```

Comment:
nit: this is sort-of buried in the middle of the script here. Maybe add a `SKIPPED_SEEDS: Final = frozenset({120})` global constant?

Reply:
Oops, I missed this suggestion... probably won't do a separate follow-up just for this but will keep it in mind when I come back to this TODO.